softicar/platform
# PLAT-648 Delete Old Program Executions Automatically

Summary:
* Add new column "executionRetentionDays" to `AGProgram` and `AGProgramLog`
* Add `ProgramExecutionsDeleter` and `ProgramExecutionsDeleterTest`
* Add new program `ProgramExecutionsCleanupProgram`

Test Plan:
* See `ProgramExecutionsDeleterTest`
* See Programs Page:
![01](https://user-images.githubusercontent.com/74466399/151322540-57cc2415-afac-49f1-aaca-b88b582c707f.png)
* See Program Executions Page:
![02](https://user-images.githubusercontent.com/74466399/151322553-25b54a09-330c-4c20-813e-0cd197966c1b.png)
